docs/api/utils.md

```diff
@@ -386,3 +386,81 @@ const countAtom = atom(0)
 `atomFrozenInDev` is another function to create a frozen atom.
 The atom is frozen only in the development mode.
 In production, it works as the normal `atom`.
+
+## splitAtom
+
+The `splitAtom` utility is useful for when you want to get an atom for each element in a list.
+It works for read/write atoms that contain a list.
+
```

Comment:
It would be nice to explain how this util works. Like, splitAtom takes an atom representing a list of items and return an array of atoms, each representing item. Maybe, typescript definition would help?

Reply:
I've updated the docs, tell me if there's anything I've missed!